app/src/main/java/ca/dal/cs/athletemonitor/athletemonitor/LoginActivity.java

```diff
@@ -0,0 +1,48 @@
+package ca.dal.cs.athletemonitor.athletemonitor;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import ca.dal.cs.athletemonitor.athletemonitor.listeners.BooleanResultListener;
+
+public class LoginActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+    }
+
+    public void signInClick(View view) throws InterruptedException {
+        //get data from activity
+        final String username = ((EditText) this.findViewById(R.id.txtUsername)).getText().toString();
+        String password = ((EditText) this.findViewById(R.id.txtPassword)).getText().toString();
+        final Activity thisActivity = this;
+
+        //attempt to authenticate
+        AccountManager.authenticate(new User(username, password), new BooleanResultListener() {
+            @Override
+            public void onResult(boolean result) {
+                Log.d("AM_DEBUG", "true login");
```

Comment:
This log message doesn't give much, maybe something like `"Authentication attempt for user " + username + " had result: " + result`

Reply:
Removed, this was an artifact.